app/(landing)/elections/position/[...loc]/components/PositionSchema.js

```diff
@@ -3,34 +3,31 @@ import { APP_BASE } from 'appEnv'
 
 export default function PositionSchema({ race, loc }) {
   const {
-    level,
+    positionLevel,
     normalizedPositionName,
     positionDescription,
-    locationName,
-    electionDay,
+    Place,
+    electionDate,
     eligibilityRequirements,
     state,
     salary,
     employmentType,
     filingOfficeAddress,
     filingPhoneNumber,
     filingDateEnd,
-    municipality,
   } = race
-  let locStr = locationName
-  if (level === 'local') {
-    locStr += `${municipality?.name}, ${state?.toUpperCase() || ''}`
+  let locStr = Place?.name || ''
+  if (positionLevel?.toLowerCase() === 'local') {
+    locStr += `, ${state?.toUpperCase() || ''}`
   }
-  if (level === 'city') {
+  if (positionLevel?.toLowerCase() === 'city') {
     locStr += ` City, ${state}`
-  } else if (level === 'county') {
-    locStr += ` County, ${race.state}`
-  } else if (level === 'state') {
-    locStr += ` ${state}`
+  } else if (positionLevel?.toLowerCase() === 'county') {
+    locStr += ` County, ${state}`
   }
   const slug = `elections/position/${loc.join('/')}`
   const url = `${APP_BASE}/${slug}`
-  const baseSalary = `${salary?.match(/\d+/g)}` || 'Not Specified'
+  const zipCode = filingOfficeAddress?.match(/\d{5}/)?.[0] || ''
```

Comment:
filingOfficeAddress starts with a five digit or greater street number 7885 times, which this will match on instead of the zip.

Try this instead
```suggestion
  const zipCode = filingOfficeAddress?.match(/\b\d{5}(?:-\d{4})?\b/)?.[0] || '';
```